packages/proto/src/messages/staking/staking.ts

```diff
@@ -0,0 +1,77 @@
+import { Coin } from '../../proto/cosmos/base/coin'
+import {
+  MsgDelegate,
+  MsgBeginRedelegate,
+  MsgUndelegate,
+} from '../../proto/cosmos/staking/tx'
```

Comment:
cancel unbond is missing

Reply:
We should separately add this to all packages in a separate PR.

Reply:
why in a separate PR? doesn't this mean that the staking proto is outdated? This should be addressed in this PR

Reply:
The staking Proto is outdated, as is the rest of the codebase. The point of this PR is to refactor the `proto` code so that it's maintainable.

When it comes to messages, we can add support across the entire codebase at once.

Reply:
sg, please create a ticket to track the task